src/adapters/envista.js

```diff
@@ -1,482 +1,167 @@
 'use strict';
 
 import { default as baseRequest } from 'request';
-import { REQUEST_TIMEOUT } from '../lib/constants';
-import cheerio from 'cheerio';
-import { default as moment } from 'moment-timezone';
-import { flattenDeep, isFinite } from 'lodash';
-import { parallel, parallelLimit, retry } from 'async';
-import { acceptableParameters, convertUnits } from '../lib/utils';
+import { REQUEST_TIMEOUT } from '../lib/constants.js';
+import { DateTime } from 'luxon';
+const { difference, flattenDeep } = pkg;
+import pkg from 'lodash';
+import { parallel, parallelLimit } from 'async';
+import { convertUnits, unifyMeasurementUnits } from '../lib/utils.js';
+
+const headers = {
+  'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0',
+  'Accept': 'text/html,application/json,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+  'Content-Type': 'text/html; charset=utf-8',
+  'envi-data-source': 'MANA',
+  'Authorization': 'ApiToken ' + '1cab20bf-0248-493d-aedc-27aa94445d15'
```

Comment:
pull the api key out as a secret so its just a variable in the file

Reply:
use environment variables like we are in the other adapters e.g. https://github.com/openaq/openaq-fetch/blob/508287ac720f06dbc0ed1295ab8e2e8c76dc4fd5/adapters/au_vic.js#L31 then we'll place is in the secrets location.

Reply:
Ok it is set as `process.env.ISRAEL_ENVISTA_TOKEN`